miso-web/src/it/resources/tomcat-config/miso.it.properties

```diff
@@ -80,6 +80,9 @@ miso.boxscanner.enabled:false
 miso.visionmate.server.host:localhost
 miso.visionmate.server.port:8000
 
+## bulk sample table settings (creation and propagation)
+miso.defaults.sample.bulk.scientificname:Homo sapiens
```

Comment:
Any reason to make this specific to the bulk page, or can we set the same default on the single page too?

Reply:
No specific reason not to extend it...